Week8_9-SQL-Optimizer/README.md

```diff
@@ -0,0 +1,4 @@
+# Week8,9 SQL Optimizer
+> 学习常见优化器架构，如 Volcano, Cascades 等，学习查询优化具体流程。实现一个并发的 MergeSort 算法，并完成性能调优。
+
+MergeSort可以完成PingCap Talent Plan里面的[MergeSort](https://github.com/IcePigZDB/talent-plan/tree/master/tidb/mergesort)。
```

Comment:
Hi, @IcePigZDB 
should we point this link to a forked-repo? Or it should be the source repository? 
if it will be good that add this link to your homework folder？when someone clicks on your repo submodule, it will help them to locate 'MergeSort'(your homework) in the whole 'talent-plan' repo.

Reply:
Format directory structure  in (#22).